wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -56,6 +58,8 @@ import           Pos.Wallet.Web.State (AddressInfo (..), AddressLookupMode (Ever
 import           Pos.Wallet.Web.Util (decodeCTypeOrFail, getAccountAddrsOrThrow,
                                       getWalletAccountIds, getWalletAddrsDetector)
 
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
Why would we want to ignore this hint here? Can you add a comment justifying it?